modules/win-autoscale/watchmaker-win-autoscale.cfn.json

```diff
@@ -780,6 +855,98 @@
                                     ]
                                 }
                             },
+                            "c:\\cfn\\scripts\\AmazonCloudWatchAgent\\aws-cloudwatch-agent-config.json": {
```

Comment:
Should be able to move this into a `files` section in the `install-cloudwatch-agent` config-block. I _think_ that may allow the removal of the `If` blocks for the ref on the log group, since that config-block only executes on the same condition.

Reply:
I moved the `files' section.  However, I left the `if` blocks since the template still errors when the log group is not present.